specification/compute/resource-manager/readme.go.md

```diff
@@ -55,6 +55,11 @@ batch:
   - tag: package-container-service-2015-11-preview
 ```
 
+```yaml $(go) && $(multiapi) && $(track2)
+batch:
```

Comment:
I thought we weren't doing batch builds for track 2?

Reply:
I am not sure how we are going to do this, but we actually do not have a way to extract the tags defined in readme.go.md for track 2. This is mainly the consideration for the SDK automation pipeline script. In a real release of track 2 modules in azure-sdk-for-go, we will not use multiapi. If the script wants to generate new packages for track 2 in a PR (like in this one), it needs to have the ability to find all tags for track 2, but currently we do not have that functionality (or we have? please let me know). One valid workaround is that we just use multiapi - autorest will help us to iterate over the tags for track 2.